src/sentry/search/snuba/executors.py

```diff
@@ -202,7 +201,10 @@ def snuba_search(
 
         conditions = []
         having = []
+        group_types = set()
         for search_filter in search_filters:
+            if search_filter.key.name in ("issue.category", "issue.type"):
+                group_types.update(value for value in search_filter.value.raw_value)
```

Comment:
Does it make sense to handle these separately?

```
if search_filter.key.name in ("issue.category"):
    group_categories.update(GroupCategory(value) for value in search_filter.value.raw_value)
if search_filter.key.name in ("issue.category", "issue.type"):
    group_categories.update(GROUP_TYPE_TO_CATEGORY[GroupType(value)] for value in search_filter.value.raw_value)
````

Reply:
Whether you query type or category, by the time it gets here it's always the type value

Reply:
Ahh, I forgot about that. We could probably use the second if and save some complexity later on.